specification/monitor/resource-manager/Microsoft.Insights/stable/2019-06-01/actionGroups_API.json

```diff
@@ -0,0 +1,879 @@
+                    "description": "The receiver status of the e-mail."
+                }
+            },
+            "required": [
+                "name",
+                "emailAddress",
+                "useCommonAlertSchema"
+            ]
+        },
+        "SmsReceiver": {
+            "description": "An SMS receiver.",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "The name of the SMS receiver. Names must be unique across all receivers within an action group."
+                },
+                "countryCode": {
+                    "type": "string",
+                    "description": "The country code of the SMS receiver."
+                },
+                "phoneNumber": {
+                    "type": "string",
+                    "description": "The phone number of the SMS receiver."
+                },
+                "status": {
+                    "readOnly": true,
+                    "$ref": "#/definitions/ReceiverStatus",
+                    "description": "The status of the receiver."
+                }
+            },
+            "required": [
+                "name",
+                "countryCode",
+                "phoneNumber"
+            ]
+        },
+        "WebhookReceiver": {
+            "description": "A webhook receiver.",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "The name of the webhook receiver. Names must be unique across all receivers within an action group."
+                },
+                "serviceUri": {
+                    "type": "string",
+                    "description": "The URI where webhooks should be sent."
+                },
+                "useCommonAlertSchema": {
+                    "type": "boolean",
+                    "description": "Indicates whether to use common alert schema."
+                },
+                "useAadAuth": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Indicates whether or not use AAD authentication."
+                },
+                "objectId": {
+                    "type": "string",
+                    "description": "Indicates the webhook app object Id for aad auth."
+                },
+                "identifierUri": {
+                    "type": "string",
+                    "description": "Indicates the identifier uri for aad auth."
+                },
+                "tenantId": {
+                    "type": "string",
+                    "description": "Indicates the tenant id for aad auth."
+                }
+            },
+            "required": [
+                "name",
+                "serviceUri",
+                "useCommonAlertSchema"
+            ]
+        },
+        "ItsmReceiver": {
+            "description": "An Itsm receiver.",
```

Comment:
nit: spell check

Reply:
This is not a new API. The changes are we are adding 4 not required fields to WebhookReceiver type.

```
- useAadAuth 
- objectId
- identifierUri
- tenantId 
```

Also the diff from previous version is 

 ```
 },
                "useAadAuth": {
                    "type": "boolean",
                    "default": false,
                    "description": "Indicates whether or not use AAD authentication."
                },
                "objectId": {
                    "type": "string",
                    "description": "Indicates the webhook app object Id for aad auth."
                },
                "identifierUri": {
                    "type": "string",
                    "description": "Indicates the identifier uri for aad auth."
                },
                "tenantId": {
                    "type": "string",
                    "description": "Indicates the tenant id for aad auth."
```

Reply:
It's much easier for reviewers when you just make the first commit an exact copy of the previous API version. Then we can just diff commit 1 with latest and it's obvious what the delta is. In any case, I confirmed this by diffing manually and changes look good.

---
In reply to: [293928750](https://github.com/Azure/azure-rest-api-specs/pull/6218#discussion_r293928750) [](ancestors = 293928750)